Packs/Legacy/Integrations/FeedTAXII/FeedTAXII.py

```diff
@@ -517,6 +521,23 @@ def __init__(self, insecure: bool = True, polling_timeout: int = 20, initial_int
                 self.username = credentials.get('identifier', None)
                 self.password = credentials.get('password', None)
 
+        if cert_text and key_text:
```

Comment:
Can this work if only one of them is inserted in some cases?
If not, we should return an error in the test stating that 2 of the params should be filled, or none of them.